src/fastjet/_generalevent.py

```diff
@@ -708,6 +710,55 @@ def exclusive_jets_softdrop_grooming(
         )
         return res
 
+    def inclusive_jets_njettiness(
+        self,
+        measure_definition="NormalizedMeasure",
+        axes_definition="OnePass_KT_Axes",
+        njets=_default_taus_njettiness,
+        beta=1.0,
+        R0=0.8,
+        Rcutoff=None,
+        nPass=None,
+        akAxesR0=None,
+    ):
+        if len(njets) == 0:
+            raise ValueError("Must provide at least one njettiness!")
+        if any(njet <= 0 for njet in njets):
+            raise ValueError("Requested njettiness must be > 0!")
```

Comment:
s/njettiness/njets/ to be consistent w/ argument naming? (also in the other files)